src/os/bluestore/BlueFS.cc

```diff
@@ -110,8 +110,10 @@ class BlueFS::SocketHook : public AdminSocketHook {
       f->dump_int("available_from_bluestore", extra_space);
       f->close_section();
     } else if (command == "bluestore bluefs stats") {
+      std::stringstream ss;
```

Comment:
i'd suggest avoid shadowing the parameter for better readability. can we use a different name?

Reply:
@tchaikov Fixed.